README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
```

Comment:
I've made some changes to this branch and would love ❤️ it if you could approve this pull request. For now, just approving the pull request will suffice. In a moment we will look at requesting changes. 👍

Reply:
...